### ISSUE TITLE
[MangaPark] Connector not working

### ISSUE THREAD

**Issue by Kurokami08** [manga-download/hakuneko#7033]
### Is there an existing issue for this connector?

- [X] I have searched the existing issues

### Did you read the troubleshooting guide ?

- [X] I read it but nothing seems related

### Is the website of the connector working properly ?

- [X] Yes i'm able to see the manga within my standard browser

### Make sure to try Nightly Build

- [X] Yes nightly build didn't fix the issue

### What kind of issue are you encountering

Manga list isn't refreshing

### Describe the bug

When attempting to download NSFW manhwa especially in MangaPark.net website, I'm encountering two recurring issues, hindering their access to desired content:

1. Missing Chapter List: Despite locating the manhwa title in the manga column, I'm unable to access the list of chapters for download. This prevents them from selecting specific chapters or tracking their reading progress effectively.

2. Incomplete Chapter Availability: Even when accessing certain manhwa titles via the app, I encounter incomplete lists of available chapters. This limitation restricts users from accessing the full range of content they expect from the platform, leading to frustration and inconvenience.

These issues not only disrupt the user experience but also diminish the platform's utility for users seeking specific NSFW manhwa content.

### To Reproduce

1. Name of connector :- Mangapark.net
2. Name of manga (if connector update is working) :- https://mangapark.net/title/358466-en-puuzaki-puuna-the-shy-snow-woman-and-the-cursed-ring-uncensored; https://mangapark.net/title/401826-en-urban-legend; https://mangapark.net/title/222563-en-concubine
3. And many more similar manhwa

**Comment by MikeZeDev** [comment]
There are multiple problems : API changes, cookie name for NSFW changed, etc..

@Druzai It seems V5 dont use "sources" for chapters anymore? Apparently i can get chapters directly now, with `comicID`

```
query Get_comicChapterList($comicId: ID!) {
  get_comicChapterList(comicId: $comicId) {
    data {
      id
      lang
      dname
      dateCreate
      sourceId
    }
  }
} 
```

What were sources in the first place? Could you enlighten me please? 

lang seems to be null on chapters :/

**Comment by MikeZeDev** [comment]
* On v.3 manga has no language attribute. 

* On v3 language is applied on chapters from the v3 endpoint when gathering chapters => We have language info at chapter level. 

* NOW :  On v5 if you have a manga in 2 language you have 2 different entries. Its easy to infer chapter language from manga, no biggie because manga language are separated.

The problem is if you fetch mangalist as V3, you dont have the language information, and therefore chapters cant inherit it.

For now users have manga list  fetched from V3, lacking the lang attribute. We should default "fetching the manga list using V5" (so we will have all the informations).

And for consistency, default everything to V5 with V3 fallbacks. Any idea against it?

**Comment by Kurokami08** [comment]
@MikeZeDev, 
Thank you for the reply, but my apologies I'm not a tech person due to which my knowledge regarding the things you mentioned above is close to 0. I'd appreciate if you could break down the process of what should I be doing or if I was not able to explain my problem that I'm facing than could you tell me how or what more details I can provide you to understand the problem in a more suitable manner.

**Comment by MikeZeDev** [comment]
TLDR : website got V3 and V5 version, and its tedious to properly handle them at the same time because there are more and more inconsistancies. And there is now a breaking change. You may have to redo all bookmarks  :/

My messages were for other devs. You have nothing to do.
